test/lender/poolAdmin.t.sol

```diff
@@ -0,0 +1,208 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.7.6;
+
+import "forge-std/Test.sol";
+
+import "src/lender/assessor.sol";
+import "src/lender/admin/pool.sol";
+import "./mock/coordinator.sol";
+import "./mock/navFeed.sol";
+import "./mock/memberlist.sol";
+import "./mock/clerk.sol";
+
+contract LiquidityManagementPoolAdminTest is DSTest {
```

Comment:
In Foundry `Test` inherits from `DSTest`. I think it might make sense to replace all `DSTest` with `Test` in this migration to have the full functionality of the foundry test base contract.

Reply:
Agreed.